frontend/src/components/pages/HomePage.js

```diff
@@ -13,7 +13,7 @@ function HomePage(props) {
     }, []);
 
     const submit = values => {
-        api.files.uploadFile(props.user, values.file).then(response => {
+        api.files.uploadFile(props.user, props.files[0].id, values.file).then(response => {
```

Comment:
Uses just the home folder because there is no way to select folder.

Reply:
If `props.files[0].id` indicates a folder, it should be renamed so.

Reply:
The above code is really confusing as now the files are returned with a list of folders, that contain files and subfolders. So some renaming is in place if we use that format to store the data in redux. I am not a the biggest fan of complex tree structures in redux store either so I am open for other suggestions.

Reply:
OK, let's discuss it today.